### PR TITLE
Creates a RAVLT cognitive task; Inlines Flanker images

### PR DESCRIPTION
**Pull request by stufreen**

### CODE REVIEW

File: activities/RAVLT/RAVLT_schema.jsonld [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  "@id": "RAVLT_schema",
+  "skos:prefLabel": "Rey Auditory Verbal Learning Test",
+  "skos:altLabel": "RAVLT",
+  "schema:description": "This activity is a demo of the Rey Auditory Verbal Learning Test cognitive task.",
+  "schema:schemaVersion": "0.0.1",
+  "schema:version": "0.0.1",
+  "preamble": "Please follow the instructions for the task.",
+  "branchLogic": {},
+  "scoringLogic": {},
+  "variableMap": [],
```
Comment (akeshavan) [inline activities/RAVLT/RAVLT_schema.jsonld]: can you add the variable map here? I know it's not being used, but I think we'll want this to be a required value eventually

File: activities/RAVLT/items/ravlt_stimulus_1.jsonld [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  "schema:description": "Instructions for the Rey Auditory Verbal Learning Test",
+  "schema:schemaVersion": "0.0.1",
+  "schema:version": "0.0.1",
+  "question": {
+    "en": "Press the Play button below to listen to a list of words."
+  },
+  "ui": {
+    "inputType": "audioStimulus"
+  },
+  "inputs": [
```
Comment (akeshavan) [inline activities/RAVLT/items/ravlt_stimulus_1.jsonld]: Instead of "inputs" it should be "responseOptions"

Comment (stufreen) [reply activities/RAVLT/items/ravlt_stimulus_1.jsonld]: I've been using `inputs` where the values relate to the configuration of the question, as opposed to the range of acceptable answers. Is this not right? My impression was that `valueConstraints` should relate to constraints on the answer the client can return.

Comment (akeshavan) [reply activities/RAVLT/items/ravlt_stimulus_1.jsonld]: maybe we should have `responseOptions` and `inputOptions` ?

**Pull request merged**